Burrows_Wheeler_Transform_Algorithm/BWT.py

```diff
@@ -0,0 +1,22 @@
+def bwt(s):
+    assert "\002" not in s and "\003" not in s, "Input string cannot contain STX and ETX characters"
+    s = "\002" + s + "\003"
+    sTable = sorted(s[i:] + s[:i] for i in range(len(s)))
+    lastColumn = [row[-1:] for row in sTable]
+    return "".join(lastColumn)
+
+def ibwt(r):
```

Comment:
What is `ibwt`?
And add some comments.

Reply:
> Remove AKS and index.md file.

@jainaman224 Can you please brief me, how should I remove the above mentioned commits?

Reply:
Please remove index.md as well.
You can ask this in public channel. So, others also get some information.